features/all-positions/AllLiquidations.tsx

```diff
@@ -211,113 +242,144 @@ const AllLiquidations = () => {
-          </div>
-          {"Page " + page + " of " + maxPage}
-          <div
-            onClick={() => {
-              setPage(page === maxPage ? page : page + 1);
-            }}
-          >
-            <Arrow faded={page === maxPage ? true : false}>→</Arrow>
-          </div>
-        </PageButtons>
-      </TableContainer>
+                </Tooltip>
+
+                <TableCell align="right"></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {reformattedLiquidationData.map((sponsor: string) => {
+                const liquidation = liquidations[sponsor];
+                return (
+                  <StyledTableRow key={sponsor}>
+                    <TableCell component="th" scope="row">
+                      <a href={getEtherscanUrl(sponsor)} target="_blank">
+                        {prettyAddress(sponsor)}
+                      </a>
+                    </TableCell>
+                    <TableCell align="left">
+                      {prettyDate(Number(liquidation.liquidationTimestamp))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {translateLiquidationStatus(
+                        Number(liquidation.liquidationTimestamp),
+                        liquidation.status
+                      )}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(liquidation.lockedCollateral))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(liquidation.tokensLiquidated))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(
+                        getDisputablePrice(liquidation.maxDisputablePrice)
+                      )}
+                    </TableCell>
+
+                    <TableCell align="right">
+                      <MoreInfo
+                        onClick={() => handleOpenActionsDialog(sponsor)}
+                      >
+                        ⋮
+                      </MoreInfo>
+                    </TableCell>
+                  </StyledTableRow>
+                );
+              })}
+            </TableBody>
+          </Table>
+          <PageButtons>
+            <div
+              onClick={() => {
+                setPage(page === 1 ? page : page - 1);
+              }}
+            >
+              <Arrow faded={page === 1 ? true : false}>←</Arrow>
+            </div>
+            {"Page " + page + " of " + maxPage}
+            <div
+              onClick={() => {
+                setPage(page === maxPage ? page : page + 1);
+              }}
+            >
+              <Arrow faded={page === maxPage ? true : false}>→</Arrow>
+            </div>
+          </PageButtons>
+        </TableContainer>
+        <LiquidationActionDialog
```

Comment:
Should we render this if `empState.isExpired`?

Reply:
ye, I've re-factored this the same way I did with the click to liquidate. The button still appears and if you click it on an expired liquidation you can still see the extra position info.

![image](https://user-images.githubusercontent.com/12886084/89872954-fd1a1880-dbb9-11ea-9970-f8b2efd194d7.png)